enterprise/api/v1/serializers.py

```diff
@@ -602,9 +602,11 @@ class EnterpriseGroupMembershipSerializer(serializers.ModelSerializer):
     """
     Serializer for EnterpriseGroupMembership model.
     """
-    learner_id = serializers.IntegerField(source='enterprise_customer_user.id', allow_null=True)
+    enterprise_customer_user_id = serializers.IntegerField(source='enterprise_customer_user.id', allow_null=True)
+    lms_user_id = serializers.IntegerField(source='enterprise_customer_user.user_id', allow_null=True)
     pending_learner_id = serializers.IntegerField(source='pending_enterprise_customer_user.id', allow_null=True)
```

Comment:
wondering if we should rename this too for consistency.  e.g. `pending_enterprise_customer_user_id`?

Reply:
Hmm to me `pending_learner_id` will only ever equal one thing (there is no notion of a pending auth user) so there's no confusion like there was with lms_user_id and ECU. However, I like consistency so I'm ok with the change